packages/react-components/src/components/Drawer/DrawerMenu.tsx

```diff
@@ -89,6 +93,13 @@ export const _DrawerMenu = (props: _DrawerMenuProps): JSX.Element => {
   return (
     <_DrawerSurface styles={props.styles?.drawerSurfaceStyles} onLightDismiss={props.onLightDismiss}>
       <Stack styles={props.styles} role="menu" data-ui-id="drawer-menu">
+        {props.title && menuItemsToRender && menuItemsToRender?.length > 0 ? (
+          <Stack horizontalAlign="center" style={{ justifyContent: 'center', height: '2rem' }}>
+            <Text color={theme.palette.neutralSecondary}>{props.title}</Text>
```

Comment:
Can we see a screenshot of this in the drawer? does the text come out light blue or something?

Reply:
That comes in another PR :)